build_defs/internal_do_not_use/j2cl_application.bzl

```diff
@@ -31,9 +35,14 @@ def j2cl_application(
         name: name of the rule.
         entry_points: JavaScript namespace of the entry point for the app.
         deps: dependencies of the app (e.g. j2cl_library, js_library, etc).
+        extra_defs: Extra flags to pass to the Closure Compiler.
+        formatting: Whether to pretty-print or minify the code.
```

Comment:
`extra_defs`: allows additional command-line arguments to be passed to the Closure Compiler
`formatting`: allows specifying pretty-print mode for debugging